tinyagentos/routes/skill_exec.py

```
@@ -457,6 +457,35 @@ async def list_tools(request: Request, agent_name: str):
+        agent = (args.get("agent_name") or "").strip()
+        if store is None or not agent:
+            return
+        files_changed = None
+        if skill_id == "file_write" and isinstance(result, dict) and not result.get("error"):
+            files_changed = [{"path": args.get("path", ""), "bytes": result.get("bytes")}]
+        # agent_name is identity, not a tool argument; keep it out of tool_args.
+        tool_args = {k: v for k, v in args.items() if k != "agent_name"}
+        asyncio.create_task(emit_tool_receipt(
+            store, agent=agent, tool_name=skill_id, args=tool_args,
+            result=result, files_changed=files_changed,
+        ))
```

Ruff (0.15.18)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)